packages/common/src/pipes/case_conversion_pipes.ts

```diff
@@ -35,7 +35,10 @@ export class LowerCasePipe implements PipeTransform {
   transform(value: string|null|undefined): string|null {
     if (value == null) return null;
     if (typeof value !== 'string') {
```

Comment:
I think we can drop the whole exception altogether in dev mode:
```suggestion
    if ((typeof ngDevMode === 'undefined' || ngDevMode) && typeof value !== 'string') {
      throw invalidPipeArgumentError(LowerCasePipe, value);
    }
```
However it's still be great to update the internal logic of the `invalidPipeArgumentError` function to use the `RuntimeError` class, something like this:
```
export function invalidPipeArgumentError(type: Type<any>, value: Object) {
  const message = `InvalidPipeArgument: '${value}' for pipe '${stringify(type)}'`;
  return new RuntimeError(RuntimeErrorCode.INVALID_PIPE_ARGUMENT, message);
}
```